docs/getting-started/using-the-block-explorer.md

```diff
@@ -4,59 +4,69 @@
-`VoteBits`               | (1) Block was approved by proof-of-stake voters. (2) Block was vetoed by proof-of-stake voters and all non-stake transactions in the block were invalidated, along with the newly generated block reward for the proof-of-work miner and the Decred Treasury.
-`Final State`            | The final state of the pseudo random number generator used for ticket selection.
-`Voters`                 | The number of successful proof-of-stake votes cast in this block. The maximum value is 5.
-`Fresh Stake`            | The number of stake ticket purchases confirmed in this block.
+`Time`                   | The time this block was created by a miner and was included in the blockchain.
+`Nonce`                  | The value used by a miner to find a valid solution for this block.
+`VoteBits`               | Used to signify the voting result on the previous block. This will either be 0 or 1. If 1, the previous block was approved by proof-of-stake voters. If 0, the previous block was disapproved by proof-of-stake voters and all non-stake transactions in the block were invalidated, including the newly generated block reward for the proof-of-work miner and the Decred Treasury.
+`Final State`            | The hash value of the final state of the lottery used to determine which tickets were eligible to vote on the previous block, and thus be included in this block. It consists of the five winning ticket hashes, as well as the state of the pseudorandom number generator that was used to select those winning tickets from the live tickets pool.
+`Voters`                 | The number of successful proof-of-stake votes cast in this block. The maximum value is 5. Tickets can fail to vote when called upon, but a minimum of 3 votes are required.
+`Fresh Stake`            | The number of stake ticket purchases in this block, which will be added to the ticket pool.
 `Revocations`            | The number of tickets that failed to vote and were revoked.
 `PoolSize`               | The total number of active proof-of-stake tickets.
-`Difficulty`             | The proof-of-work network difficulty.
-`SBits`                  | The price of one proof-of-stake ticket.
 `Bits`                   | A compact version of the network difficulty at the time the block was mined.
-`Size`                   | The size of the block (in bytes).
-`Version`                | The version of the block.
-`Nonce`                  | The value used by a miner to find the correct solution for this block.
+`SBits`                  | The price of one proof-of-stake ticket.
+`Stake Version`          | The stake version used for voting on the created block.
+`Difficulty`             | The proof-of-work network difficulty.
+
+For a more advanced description of block headers, see the Dev Docs on [Block Header Specifications](https://devdocs.decred.org/developer-guides/block-header-specifications/).
```

Comment:
I think this line would be better above the table. Its easy to miss at the bottom.